std/algorithm/comparison.d

```diff
@@ -619,6 +619,28 @@ Note:
 auto cmp(R1, R2)(R1 r1, R2 r2)
 if (isInputRange!R1 && isInputRange!R2)
 {
+    static if (isDynamicArray!R1 && isDynamicArray!R2 &&
+        is(R1 : E1[], E1) && is(R2 : E2[], E2) // just to get E1 & E2
+        && __traits(isUnsigned, E1) && __traits(isUnsigned, E2)
+        && E1.sizeof == E2.sizeof && !is(E1 == __vector) && !is(E2 == __vector))
```

Comment:
```console
Running DScanner
generated/dscanner-e4ee46366d21eaa613da13c52f3f1a66684cdbaa/dsc --config .dscanner.ini --styleCheck etc std -I.
std/algorithm/comparison.d(625:56)[error]: Expected `(` instead of `)`
std/algorithm/comparison.d(625:56)[error]: Primary expression expected
```
This is complaining about `!is(E1 == __vector)`. It looks like DScanner needs to be fixed.

Reply:
https://github.com/dlang-community/D-Scanner/issues/716